interface/main/tabs/templates/tabs_template.php

```diff
@@ -38,6 +38,7 @@
 
         <!-- ko  foreach: tabs.tabsList -->
         <div class="frameDisplay" data-bind="visible:visible">
+            <h5 class="text-center position-relative" style="top:50%;" data-bind="text: loading_text,visible:loader"></h5>
```

Comment:
data-bind="text is probably okay but we are currently passing html so let's keep consistent.